templates/wordpress/files/README.md

```diff
@@ -59,3 +59,9 @@ The following changes have been made relative to WordPress as it is downloaded f
 * [WordPress](https://wordpress.org/)
 * [WordPress on Platform.sh](https://docs.platform.sh/frameworks/wordpress.html)
 * [PHP on Platform.sh](https://docs.platform.sh/languages/php.html)
+
+### Getting Started
+
+To help you get started customizing Wordpress on Platform.sh, several guides and how-to's are available on our [Community Site](https://community.platform.sh).
+- [How to install Wordpress plugins and themes with Composer](https://community.platform.sh/t/how-to-install-wordpress-plugins-and-themes-with-composer/507)
+- [How to install custom/private Wordpress plugins and themes with Composer](https://community.platform.sh/t/how-to-install-custom-private-wordpress-plugins-and-themes-with-composer/622)
```

Comment:
Bullet points should use *, not -, for consistency.

If we're going to include these, shouldn't they go above References?  Or perhaps as a section within References?  I don't see how a second list is helpful, but I could see including these in the References link list that's already there.